package.json

```diff
@@ -52,7 +52,10 @@
     "ssl/"
   ],
   "scripts": {
-    "prepublish": "webpack ./client/live.js client/live.bundle.js --color --config client/webpack.config.js -p && webpack ./client/index.js client/index.bundle.js --color --config client/webpack.config.js -p",
+    "prepublish": "npm run webpack-live && npm run webpack-index && npm run webpack-sockjs",
+    "webpack-live": "webpack ./client/live.js client/live.bundle.js --color --config client/webpack.config.js -p",
```

Comment:
Can you change `webpack-` to `client-` here? `webpack` doesn't tell me anything in this case.